spec/system/candidate_signs_up_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+describe 'A candidate signs up to Apply service' do
+  context 'when a new candidate tries to sign in' do
+    before do
+      visit '/'
+      click_on t('application_form.begin_button')
+      fill_in t('authentication.sign_up.email_address.label'), with: 'april@pawnee.com'
+      click_on t('authentication.sign_up.button')
+    end
+
+    it 'can allow a candidate to submit their email' do
+      expect(page).to have_content t('authentication.check_your_email')
+    end
+
+    it 'can sign in a candidate using the magic link' do
```

Comment:
Maybe smush this spec with the spec above? It's all part of one action and being a system spec the setup is quite heavy